File: Miranda/FizzBuzz/fizzbuzz.m
run n = concat (map fizzbuzz [1..n])
fizzbuzz i = "FizzBuzz\n", if i mod 15 == 0
           = "Fizz\n", if i mod 3 == 0
           = "Buzz\n", if i mod 5 == 0
           = show i ++ "\n", otherwise

File: VisualBasic/FizzBuzz/fizzbuzz.vb
Module Program
    Function fizzbuzz(i As Integer) As String
        If i Mod 15 = 0 Then
            Return "FizzBuzz"
        ElseIf i Mod 3 = 0 Then
            Return "Fizz"
        ElseIf i Mod 5 = 0 Then
            Return "Buzz"
        Else
            Return i.ToString
        End If
    End Function

    Sub Run(n As Integer)
        For i As Integer = 1 To n
            Console.WriteLine(fizzbuzz(i))
        Next
    End Sub

    Sub Main(args As String())
        Run(15)
    End Sub
End Module
